test/runnable/test22974.c

```diff
@@ -0,0 +1,12 @@
+// REQUIRED_ARGS: imports/test22974b.c
```

Comment:
```suggestion
// EXTRA_FILES: imports/test22974b.c
```

Reply:
I tried that. Didn't work, either, as the tester wouldn't put it on the command line.

Reply:
Did you mix up `EXTRA_SOURCES` with `EXTRA_FILES`, (like I did before I edited my comment)? `EXTRA_SOURCES` works locally for me, while `EXTRA_FILES` doesn't because it is ignored by the test runner. (Apparently it only "serves as documentation of the test itself")

Reply:
`EXTRA_SOURCES: imports/test22974b.c` works as expected.

> Apparently it only "serves as documentation of the test itself"

For dmd. GDC also relies on this directive to copy files if necessary IIRC

Reply:
Thanks for the help!